example/src/generated/modules.ts

```diff
@@ -12,7 +12,15 @@ interface NativeE2ETestInterface {
 
   testFlowComplex: Next<com.myrnproject.shared.Test>;
 
-  testFlowType: Next1<com.myrnproject.shared.FlowTest, number>;
+  testFlowParameterized: Next1<com.myrnproject.shared.FlowTest, number>;
+
+  testFlowParameterized2: Next2<com.myrnproject.shared.FlowTest, number, string>;
+
+  testFlowParameterizedComplex: Next1<com.myrnproject.shared.FlowTest, com.myrnproject.shared.Test>;
```

Comment:
Shouldn't this be string as the second argument, because this expects the argument to already be serialized?

Reply:
No this are the types from the perspective of the caller of useFlow who wants to pass his complex argument as is, not as a seralized string.

Reply:
No this is the native view which is already serialized, see the interface name. Below is the caller facing interface.

Reply:
Oh I see! This is a whole nother issue! I'll fix it

Reply:
fixed it